docker/README.md

```diff
@@ -127,13 +127,20 @@ docker build -t iot-agent . --build-arg DOWNLOAD=stable
 docker build -t iot-agent . --build-arg DOWNLOAD=1.7.0
 ```
 
--   To download code from your own fork of the GitHub repository add the `GITHUB_ACCOUNT` and `GITHUB_REPOSITORY`
```

Comment:
It seems the changes in this file has not been fully synced with https://github.com/telefonicaid/iotagent-json/pull/390 (for instance, I'm still seeing "How to build your own image" instead of "How to build an image").

Reply:
Fixed 604b2b9 - Must have missed that one.